tests/unit/conftest.py

```diff
@@ -58,6 +58,26 @@ def project_with_datasets(project):
     return project
 
 
+@pytest.fixture()
```

Comment:
We have not done it for `project/datasets` but is it worth adding a fixture/tests for orphaned plates in addition to plates in screens?